frontend/js/pages/Home.tsx

```diff
@@ -1,20 +1,20 @@
 import { useState, useEffect } from "react";
 import Button from "react-bootstrap/Button";
-import { useDispatch, useSelector } from "react-redux";
 
 import DjangoImgSrc from "../../assets/images/django-logo-negative.png";
-import { AppDispatch, RootState } from "../store";
-import { fetchRestCheck } from "../store/rest_check";
```

Comment:
@fjsj Let me know if you believe we should remove those files.

Reply:
Yes, please do.

Reply:
Done (commit 66dd971)